back/test/data/beneficiary.py

```diff
@@ -109,6 +109,23 @@ def another_male_beneficiary_data():
     }
 
 
+# def recent_beneficiary_data():
```

Comment:
I'd actually like to see a recent beneficiary in the test data (maybe registered in the previous month?) to make the test more reliable.
Is there a particular reason why you commented this?